docs/content/how-to/aws/create-aws-hosted-cluster-arm-workers.md

```diff
@@ -1,10 +1,66 @@
 ---
-title: Create ARM NodePools on AWS Hosted Cluster
+title: Create Arm NodePools on AWS Hosted Clusters
 ---
 
-# Create ARM NodePools on AWS Hosted Cluster
+# Create Arm NodePools on AWS Hosted Clusters
 
-## Creating a Hosted Cluster with ARM NodePool
+The `arch` flag was added to the NodePool Spec in OCP 4.14. The `arch` flag sets the preferred processor architecture for the NodePool (currently only supported on AWS).
+
+!!! note 
+
+    Currently, the only valid values for '--arch' are 'arm64' and 'amd64'. The HyperShift CLI will default to 'amd64' when the 'arch' flag is not specified by the user.
+
+## Creating Arm NodePools Through the API
+The hosted cluster custom resource (CR) must utilize a multi-arch manifested image. Here is an example with an OCP 4.14 multi-arch nightly image:
+```
+apiVersion: hypershift.openshift.io/v1beta1
+kind: HostedCluster
+metadata:
+  creationTimestamp: null
+  name: hypershift-arm
+  namespace: clusters
+spec:
+  autoscaling: {}
+...
+  release:
+    image: quay.io/openshift-release-dev/ocp-release-nightly@sha256:390a33cebc940912a201a35ca03927ae5b058fbdae9626f7f4679786cab4fb1c
```

Comment:
I not sure this example really shows anything of value since it isn't obvious what about the image makes it multi-arch here, other than it being a multi nightly.

If we did a `skopeo inpsect` to show what makes the image multi-arch i.e. the manifest list, that would be useful.

Also might want to link where multi nightlies can be found https://multi.ocp.releases.ci.openshift.org/

Reply:
+1